src/views/forms/NewMarketForm/v7/NewMarketPreviewStep.tsx

```diff
@@ -44,32 +49,46 @@ export const NewMarketPreviewStep = ({
   shouldHideTitleAndDescription,
 }: NewMarketPreviewStepProps) => {
   const stringGetter = useStringGetter();
-  const [errorMessage, setErrorMessage] = useState();
+  const [errorMessage, setErrorMessage] = useState<string>();
   const [hasAcceptedTerms, setHasAcceptedTerms] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   const [showAgreement, setShowAgreement] = useState(false);
   const baseAsset = getDisplayableAssetFromTicker(ticker);
   const launchableAsset = useMetadataServiceAssetFromId(ticker);
   const { createPermissionlessMarket } = useSubaccount();
   const { usdcImage } = useTokenConfigs();
+  const { freeCollateral } = useAppSelector(selectSubaccountStateForVaults);
+
+  const { alertInfo, shouldDisableForm } = useMemo(() => {
+    let alert: { type: AlertType; message: string } | null = null;
+    let shouldDisable: boolean = false;
 
-  const alertMessage = useMemo(() => {
-    let alert;
     if (errorMessage) {
       alert = {
         type: AlertType.Error,
         message: errorMessage,
       };
-    }
+    } else if (MustBigNumber(freeCollateral).lt(DEFAULT_VAULT_DEPOSIT_FOR_LAUNCH)) {
+      alert = {
+        type: AlertType.Error,
+        message: stringGetter({
+          key: STRING_KEYS.LAUNCHING_MARKET_REQUIRES_USDC,
+          params: {
+            USDC_AMOUNT: DEFAULT_VAULT_DEPOSIT_FOR_LAUNCH,
+          },
+        }),
+      };
 
-    if (alert) {
-      <AlertMessage type={alert.type}>{alert.message} </AlertMessage>;
+      shouldDisable = true;
     }
 
-    return null;
-  }, [errorMessage]);
+    return {
+      alertInfo: alert,
+      shouldDisableForm: shouldDisable,
```

Comment:
could this just be shouldDisableForm: !!alert 

?

Reply:
If it is an error due to tx we don't want to disable the form but we still want to display the error

Reply:
oo the diff was very hard to read, my bad